asv.conf.json

```diff
@@ -4,8 +4,12 @@
     "project_url": "https://www.djangoproject.com/",
     "repo": "https://github.com/django/django.git",
     "branches": ["main"],
+    "build_command": [
+        "python -m pip install build",
+        "python -m build --wheel -o {build_cache_dir} {build_dir}"
+    ],
     "environment_type": "conda",
     "conda_channels": ["conda-forge", "defaults"],
     "show_commit_url": "http://github.com/django/django/commit/",
-    "pythons": ["3.10", "3.9", "3.8"]
+    "pythons": ["3.10"]
```

Comment:
@smithdc1 so I ideally would like us to run benchmarks for 3.10, 3.11 and 3.12
@smithdc1 Will there be any issues with me updating to this do you think?

Reply:
I think adding the new versions should "just work" (tm).

Reply:
🤞 added in